src/css/utilities.typography.css

```diff
@@ -56,6 +56,14 @@
   line-height: var(--font-leading-normal);
 }
 
+.text-leading-8 {
```

Comment:
Curious about these leading-8/10s. Is this mixing the concept of _size_ and _type_? What scenarios might we need this over our baseline definition of line-heights?

Reply:
Ah, these were supposed to be removed as they're not needed.